app/controllers/claims/schools/users_controller.rb

```diff
@@ -18,12 +18,9 @@ def check
   end
 
   def create
-    if user_form.invite
-      redirect_to claims_school_users_path(@school)
-      flash[:success] = t(".user_added")
-    else
-      render :new
-    end
+    user_form.invite!
+    redirect_to claims_school_users_path(@school)
+    flash[:success] = t(".user_added")
```

Comment:
Nitpick and personal preference,

Does inlining these read better for you?

```suggestion
    redirect_to claims_school_users_path(@school), flash: { success: t(".user_added") }
```

This helps me see the flash as data added to the redirection.

Reply:
I couldn't care if I tried. Happy to change it if it makes more sense like this to other people.